core/flyout_horizontal.js

```diff
@@ -275,6 +321,22 @@ Blockly.HorizontalFlyout.prototype.wheel_ = function(e) {
 };
 
 /**
+ * Select a category using the scroll position
+ * @param  {number} pos The scroll position.
+ */
+Blockly.HorizontalFlyout.prototype.selectCategoryByScrollPosition = function(pos) {
+  var scaledPos = pos / this.workspace_.scale;
+  // Traverse the array of scroll positions in reverse, so we can select the lowest
```

Comment:
Update this to refer to "before", "past", and "furthest" (or similar terms) so that it's not specific to horizontal or vertical.

Reply:
all set